lib/MusicBrainz/Server/Data/Editor.pm

```diff
@@ -186,8 +186,13 @@ sub find_by_ip {
 sub search_by_email {
     my ($self, $email) = @_;
 
+    # Remove periods (\.), and anything preceded by a + sign for email matching,
+    # since both of these are often used as "free aliases" by email providers
+    $email =~ s/(\\.|\+[\d]+)//g;
```

Comment:
* It should probably affect the user info only.
* `+` sign should be removed too if followed by nothing or by non-digits.

```suggestion
    $email =~ s/\+.*@/@/;
    $email =~ s/\\.(.*@)/$1/g;
```

The above comment should updated accordingly.